templates/image/README.md

```
@@ -0,0 +1,57 @@
+
+```shell
+nix build .#image
+docker load < result
+docker run -p 8080:8080 ping-server:latest
+```
+
+> [!NOTE]
+> On macOS, the image is built for Linux via the
+> [linux-builder](https://nixos.org/manual/nixpkgs/stable/#sec-darwin-builder). Follow the
+> setup guide to configure Nix, then spin up the VM before building:
+> ```shell
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)